EIPS/eip-4762.md

```diff
@@ -258,10 +258,6 @@ When (and only when) calling a system contract either
 
 Also corresponding witness costs need to be charged for _precompile/opcode resolution_ but are not charged in the _system call_.
 
-### Account abstraction
-
-TODO : still waiting on a final decision between 7702 and 3074
```

Comment:
while this may be outdated, one needs to come up with any considerations for account abstraction w.r.t. 7702. till then this section serves as placement